.github/actions/install-deps-action/action.yml

```diff
@@ -16,30 +16,11 @@ runs:
     ### DOWNLOAD AND INSTALL DEPENDENCIES ###
 
     # Download dependencies packaged by Ubuntu
-    - run: sudo apt -y install bison busybox-static cargo cmake coreutils cpio elfutils file flex gcc gcc-multilib git iproute2 jq kbd kmod libcap-dev libelf-dev libunwind-dev libvirt-clients libzstd-dev linux-headers-generic linux-tools-common linux-tools-generic make ninja-build pahole pkg-config python3-dev python3-pip python3-requests qemu-kvm rsync rustc stress-ng udev zstd libseccomp-dev libcap-ng-dev
+    - run: sudo apt -y install bison busybox-static cargo cmake coreutils cpio elfutils file flex gcc gcc-multilib git iproute2 jq kbd kmod libcap-dev libelf-dev libunwind-dev libvirt-clients libzstd-dev linux-headers-generic linux-tools-common linux-tools-generic make ninja-build pahole pkg-config python3-dev python3-pip python3-requests qemu-kvm rsync rustc stress-ng udev zstd libseccomp-dev libcap-ng-dev llvm clang python3-full pipx curl 
       shell: bash
 
-    # clang 17
-    # Use a custom llvm.sh script which includes the -y flag for
-    # add-apt-repository. Otherwise, the CI job will hang. If and when
-    # https://github.com/opencollab/llvm-jenkins.debian.net/pull/26 is
-    # merged, we can go back to using https://apt.llvm.org/llvm.sh.
-    - run: wget https://raw.githubusercontent.com/Decave/llvm-jenkins.debian.net/fix_llvmsh/llvm.sh
-      shell: bash
-    - run: chmod +x llvm.sh
-      shell: bash
-    - run: sudo ./llvm.sh all
-      shell: bash
-    - run: sudo ln -sf /usr/bin/clang-17 /usr/bin/clang
-      shell: bash
-    - run: sudo ln -sf /usr/bin/llvm-strip-17 /usr/bin/llvm-strip
-      shell: bash
-    # meson
-    - run: pip install meson
-      shell: bash
-
-    # Install virtme-ng
-    - run: pip install virtme-ng
+    # meson and virtme-ng
+    - run: pip3 install meson virtme-ng --break-system-packages
```

Comment:
Should we apt install meson? Is there any reason to have it installed via pip?

I'd like to do this for virtme-ng as well, but at the moment the version from pip is much more stable, once it's fixed in Ubuntu we can also move virtme-ng to the list of packages installed via apt install.

Reply:
> Should we apt install meson? Is there any reason to have it installed via pip?

None I'm aware of provided the build passes and works. 24.04 has 1.3.2 and pip has 1.5.1.